place/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls    import path
+from .views         import PlaceView
+
+urlpatterns =[
+    path('placeview/',PlaceView.as_view()),
```

Comment:
```suggestion
    path('/placeview',PlaceView.as_view()),
```
app url이 끝나는 지점에는 슬래시를 붙이지 않습니다 :)